lib/node_modules/@stdlib/blas/base/gscal/docs/types/index.d.ts

```diff
@@ -32,12 +32,12 @@ type InputArray = NumericArray | Collection<number> | AccessorArrayLike<number>;
 */
 interface Routine {
 	/**
-	* Multiplies a vector `x` by a constant `alpha`.
+	* Multiplies a vector `x` by a scalar constant `alpha`.
 	*
 	* @param N - number of indexed elements
-	* @param alpha - constant
+	* @param alpha - scalar constant
 	* @param x - input array
-	* @param stride - stride length
+	* @param strideX - stride length
```

Comment:
```suggestion
	* @param stride - stride length
```
Same suggestion throughout this file.